wifiphisher/pywifiphisher.py

```diff
@@ -420,6 +430,15 @@ def start(self):
         try:
             if self.opmode.internet_sharing_enabled():
                 self.network_manager.internet_access_enable = True
+                # Set up an automatic MITM attack if `-m/--mitminterface` is present. 
+                #
+                # We are already handling the chosen interface as an internetInterface.
+                # Here we are also protecting the rest of the detected interfaces.
+                #  (i.e. prevent NetworkManager from managing them)
+                if args.mitminterface:
+                    restoftheinterfaces=interfaces.NetworkManager._name_to_object.items()
```

Comment:
I'm getting: "AttributeError: type object 'Network Manager' has no attribute '_name_to_object'."

Why are you not using the singleton object "network_manager" (defined on L332) to get the rest of the interfaces?

Reply:
My bad, I missed the singleton, although it was used everywhere else. I changed it now.